src/aics-image-viewer/components/App/index.tsx

```diff
@@ -960,6 +968,11 @@ export default class App extends React.Component<AppProps, AppState> {
     this.handleChangeToImage(SHOW_AXES, showAxes);
   }
 
+  changeBoundingBoxShowing(showBoundingBox) {
+    this.setUserSelectionsInState({ showBoundingBox });
+    this.handleChangeToImage("showBoundingBox", showBoundingBox);
```

Comment:
ah, now this part is awkward.  We are using this string only as a key into the switch statement in handleChangeToImage. (which really is more like a map of string to functioncall, which could be done differently than a switch statement).    So the question in my mind is do we just make calls on view3D from here directly, or try to keep it abstracted in some way.   I feel like it would not be so bad to just make the call directly on view3d.

Reply:
Looks like a benefit of `handleChangeToImage` is it puts the check that `view3d` and `image` are defined in one place (line 767) rather than spread wherever a call to `view3d` is made - are there cases in which either would be missing outside startup?

Reply:
I see your point. It's not a huge deal but it doesn't look good when it's a quoted string like that.  After #32 they are all going to become quoted strings in there.  There might be a cleaner way (as part of implementing #32) to make it an object with properties like:
```
{
  showBoundingBox: ()=>view3d.setShowBoundingBox(...),
}
```
That's very bad pseudo code and leaving out important details but at least it doesn't use string literals.